body/mtv/pcb/README.md

```diff
@@ -0,0 +1,61 @@
+## Electronics and Control Development 
+
+### Control architecture overview of the MTV
+
+The MTV control architecture is of cascaded type. The high level commands are computed on a smartphone, running the Intel [OpenBot](https://www.openbot.org/) framework. These commands are then passed to a ESP32 low-level control board which handles PWM generation as well as communication with the different MTV sensors (e.g. encoders, sonars and so on).
+
+<p align="center">
+  <img src="../../../docs/images/MTV/Ctrl_arch.png" width="800" alt="App GUI"/>
+</p>
+
+The ESP32 low-level control board communicates with the smartphone through a dedicated serial-USB interface. The command strings have the following structure: 
+
+- | **Drive letter** | **LHS speed** | **RHS speed** |
```

Comment:
This section here should probably be removed. We will control it with our OpenBot firmware. But the description of this should probably be in the main README file.